### PR TITLE
.gitignore `make test`-generated files

### PR DESCRIPTION
**Pull request by lawcho**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,6 +57,12 @@
 /test/Compiler/with-stdlib/AllStdLib
 /test/Compiler/with-stdlib/agda-rts.js
 /test/Compiler/with-stdlib/highlight-hover.js
+/test/interaction/*.out
```
Comment (andreasabel) [inline .gitignore]: Not these.  These are our golden values. We track the output produced by Agda in these files.

**Review by andreasabel — CHANGES_REQUESTED** [review]
Thanks.

File: .gitignore (hunk 1 above)
Comment (lawcho) [reply .gitignore]: Checked the files in instead

**Review by andreasabel — APPROVED** [review]

**Comment by andreasabel** [comment]
Sorry, the files `/test/interaction/Issue6261?.out` are indeed to be ignored. Fix in https://github.com/agda/agda/commit/aecfea06b9564d416ecc6e4bc474c2119e97943f.

**Pull request merged**